### DISCUSSION TITLE
SumatraPDF should be responding even with issues at PDFs

### DISCUSSION THREAD

**Discussion by koppor** [sumatrapdfreader/sumatrapdf#4119] [General]
I tried to open a PDF and following was the UI state then:

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/1366654/d4b800c1-efd4-4d9f-afde-bf142528f7d9)

I closed and re-opened SumatraPDF and no result...

    sumatrapdf --help

Started SumatraPDF again. I could not stop it from opening old files. SumatraPDF should have a timeout of 1 minute if some task hangs and kill it.

---

Task manager:

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/1366654/40d22a70-4176-4893-b884-d667f6c07245)

I am even unable to use "End task" of Windows Task Manager.

**Comment by GitHubRulesOK** [comment]
1) Check in current [Pre-Release](https://www.sumatrapdfreader.org/prerelease) as such errors (if known) should be trapped.
2) Provide a sample of a problem file as without analysis as to "why it hangs" there cannot be a fix. (if the file is not to be public property then email to the developer)

**Comment by koppor** [comment]
I'll close. I will reopen if I have a reproducer...
